src/main/resources/static/index.html

```diff
@@ -1,14 +1,132 @@
-	</head>
-	<body onload="checkToken()">
-		<div class="main-page">
-		      <p class="message">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla a sagittis magna. Vivamus auctor aliquam mauris vitae malesuada. Maecenas pulvinar ac velit vel auctor. Etiam nec tellus a est imperdiet dignissim quis nec metus. Proin ultrices consectetur sem. Donec urna massa, fermentum non metus nec, varius pulvinar ipsum. Donec vel tellus egestas, vestibulum dolor a, convallis dolor. Curabitur quis elementum tortor. In commodo turpis id facilisis convallis. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed interdum sodales eros ut hendrerit. Praesent laoreet nibh eget enim cursus, at viverra arcu elementum. Cras vestibulum libero justo, imperdiet rutrum nunc sollicitudin non.</p>
-		</div>
-	</body>
+
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css"
+        integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/override-style.css">
+
+    <script src="scripts/checkToken.js"></script>
+    <script src="scripts/homePage.js"></script>
+    <script src="scripts/alert.js"></script>
+    <script src="scripts/refreshToken.js"></script>
+
+    <title>Dashboard - Shared Rent</title>
+
+</head>
+
+<body>
+
+    <!-- Navigation -->
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark static-top">
+        <div class="container">
+            <a class="navbar-brand" href="./index.html">Shared Rent</a>
+            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive"
+                aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse" id="navbarResponsive">
+                <ul class="navbar-nav ml-auto">
+                    <li class="nav-item active">
+                        <a class="nav-link" href="./index.html">Dashboard
+                            <span class="sr-only">(current)</span>
+                        </a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="./administration.html">Administration</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">ToDo1</a>
+                    </li>
+                    <li class="nav-item">
+                        <a class="nav-link" href="#">ToDo2</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </nav>
+
+    <!-- Page Content -->
+    <div class="container">
+
+        <div id="alerts">
+            <!-- EXAMPLE
```

Comment:
Same in this file (commented code)

Reply:
I am leaving this code as this is a lot more clear for me how the whole element looks like after creation. It is a bit messy to analyze in JS scripts.